packages/kuma-gui/src/app/x/components/x-teleport/XTeleportTemplate.vue

```diff
@@ -7,19 +7,30 @@
   </Teleport>
 </template>
 <script lang="ts" setup>
-import { onMounted, ref } from 'vue'
+import { onMounted, onBeforeUnmount, ref } from 'vue'
 const props = defineProps<{
   to: {
     name: string
   }
 }>()
-
+const controller = new AbortController()
 const ready = ref<boolean>(false)
+
+const isCustomEvent = (e: Event | CustomEvent): e is CustomEvent => 'detail' in e
+
```

Comment:
Sorry, could not add code suggestion over untouched lines 😓 

```suggestion
```

Reply:
Oh 🤦 , one sec